### PR TITLE
Inital ci yarn test

### PR DESCRIPTION
**Pull request by AloeareV**
Let's see if this CI works. 

Edit: CI is working!

### CODE REVIEW

File: __tests__/__snapshots__/About.snapshot.tsx.snap [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        style={
+          Object {
+            "color": "rgb(28, 28, 30)",
+            "opacity": 0.65,
+          }
+        }
+      >
+        
+
+        Copyright (c) 2018-2021, Aditya Kulkarni.
```
Comment (zancas) [inline __tests__/__snapshots__/About.snapshot.tsx.snap:90]: This...  seems out of date?

File: __tests__/__snapshots__/About.snapshot.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        
+
+        
+
+        Built with React Native.
+        
+
+        
+
+        The MIT License (MIT) Copyright (c) 2018-2021 Zecwallet
```
Comment (zancas) [inline __tests__/__snapshots__/About.snapshot.tsx.snap:100]: Maybe the "zingolabs developers (c) 2022" should be a thing?

Comment (juanky201271) [reply __tests__/__snapshots__/About.snapshot.tsx.snap:100]: It would be a good idea to change it for that... if we are sure about 'copyright' thingy.

Comment (juanky201271) [reply __tests__/__snapshots__/About.snapshot.tsx.snap:100]: But this text of About screen, has nothing to do with this PR and CI

**Review by juanky201271 — APPROVED** [review]
It looks perfect.

File: __tests__/__snapshots__/About.snapshot.tsx.snap (hunk 1 above)
Comment (AloeareV) [reply __tests__/__snapshots__/About.snapshot.tsx.snap:90]: Agreed. Also agree with Juanky that this is out of scope, these snapshots are procedurally generated by the snapshot tests.

**Pull request merged**